calcom/cal.com
# fix: strip markdown from meta description tag in users page

## What does this PR do?

remove markdown from description meta tags in user page and also closes  #8745

Fixes 

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

1. add a markdown content to your bio
2. go to your public profile page
3. check the description html meta tag, markdown should be stripped
